internal/node/node.go

```diff
@@ -120,6 +122,7 @@ func (n *KoinosP2PNode) handleBlockBroadcast(topic string, data []byte) {
 	binary := types.NewVariableBlob()
 	binary = blockBroadcast.Serialize(binary)
 	n.Gossip.Block.PublishMessage(context.Background(), binary)
+	log.Infof("Publishing Block: %s", util.BlockString(&blockBroadcast.Block))
```

Comment:
Can we make this `Publishing block: ` to be consistent with other microservice logs?